lib/smart_answer/calculators/state_pension_age_calculator.rb

```diff
@@ -16,6 +16,19 @@ def state_pension_date
       StatePensionDateQuery.state_pension_date(dob, gender)
     end
 
+    def approaching_pension_age?
```

Comment:
I think the fact that they are approaching pension age is the reason why they are allowed to claim. So I think I would prefer if this method was named something more like `can_apply?`.